src/main/java/io/fusionauth/domain/FamilyConfiguration.java

```diff
@@ -64,6 +65,11 @@ public FamilyConfiguration(FamilyConfiguration other) {
     this.parentRegistrationEmailTemplateId = other.parentRegistrationEmailTemplateId;
   }
 
+  @JsonIgnore
+  public boolean checkChildAge() {
+    return !allowChildRegistrations || parentEmailRequired;
+  }
+
```

Comment:
If `familyConfigurat.enabled = true`, this is really the main check we use for validation on applications, tenants, and forms.  I wanted to encapsulate this in one method so if anything ever does change, we don't need to change it everywhere.

Reply:
If this is used in validation or other internal services, this logic may not belong in the public java-client.